data/human/news.txt

```diff
@@ -225,6 +230,10 @@ news "factory worker"
 			"Some of the stuff we make here ends up as one little part of a spaceship. Whenever you see a ship, always remember that it represents the effort of over a thousand of us workers."
 			"When things go wrong, somebody is responsible, and somebody gets blamed. Those somebodies are rarely the same person, unfortunately."
 			"Some people claim our universe is a simulation, but my job is so boring that I'm sure nobody would ever want to simulate it."
+			"I do a parkour over the buildings to get to work, because the ground transports are just overcrowded, and the waiting times for shuttle transports are horrible."
+			"I love working at the syndicate. The small company where I worked before didn't have showers, all people that came to office by foot were smelling bad throughout the day."
+			"Moving to this planet was the best thing I did; My commute changed from 3 hours spaceflight to just 15 minutes walking."
```

Comment:
Three hours in spaceflight? Going to another system takes a day in hyperspace.
```suggestion
			"Moving to this planet was the best thing I did; my commute changed from 3 hours spaceflight to just 15 minutes walking."
```

Reply:
Perhaps "by shuttle" instead?